arches_modular_reports/src/arches_modular_reports/ModularReport/components/ChildTile.vue

```diff
@@ -68,9 +68,7 @@ function isTileorTiles(input: unknown) {
 }
 
 function nodeValueIsEmpty(nodeValue: NodeData | NodegroupData | null) {
-    return (
-        nodeValue === null || (nodeValue as NodeData).interchange_value === null
-    );
+    return nodeValue === null || (nodeValue as NodeData).node_value === null;
```

Comment:
is this possible?  will `nodeValue` ever equal `null` now that we're marrying to the querysets API?

Reply:
(The value of a cardinality-1 nodegroup alias can be null.)